### ISSUE TITLE
Incompatible with Rails 4

### ISSUE THREAD

**Issue by pedrogrande** [Octo-Labs/easymarklet#1]
Just tried bundling easymarklet in my Rails 4 app and errored for being incompatible.

```
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    easymarklet (>= 0) ruby depends on
      rails (~> 3.2.8) ruby

    rails (4.0.2)
```

**Comment by davedawson** [comment]
Getting the same issue here. Would love to be able to use this in my Rails 4 app.

**Comment by jagthedrummer** [comment]
I just updated the `.gemspec` to allow rails `>= 3.2.9` and released version `0.0.12` (https://rubygems.org/gems/easymarklet/versions/0.0.12).  I didn't have time to test it on Rails 4, but that should at least allow you to install it.  Please let me know if you run into trouble with it on Rails 4.

**Comment by davedawson** [comment]
Beautiful. Looks like it works now. Thanks!
